microsoft-edge/devtools-guide-chromium/whats-new/2020/11/devtools.md

```diff
@@ -32,7 +32,7 @@ ms.date: 05/04/2021
 <!-- Title: Microsoft Edge and Microsoft Edge WebDriver on Linux  -->
 <!-- Subtitle: Get Microsoft Edge Dev on Ubuntu, Debian, Fedora, and openSUSE distributions and start automating in CI/CD environments with Microsoft Edge WebDriver. -->
 
-Microsoft Edge Dev is now supported on Ubuntu, Debian, Fedora, and openSUSE distributions.  Download and install the Microsoft Edge Dev `.deb` or `.rpm` package directly from the [Microsoft Edge Insider site](https://www.microsoftedgeinsider.com/download?platform=linux) or use the standard package management tools of your Linux distribution.
+Microsoft Edge Dev is now supported on Ubuntu, Debian, Fedora, and openSUSE distributions.  Download and install the Microsoft Edge Dev `.deb` or `.rpm` package directly from the [Microsoft Edge Insider site](https://www.microsoft.com/edge/download/insider?platform=linux) or use the standard package management tools of your Linux distribution.
```

Comment:
orig url:
https://www.microsoftedgeinsider.com/download?platform=linux
goes to:
https://www.microsoft.com/en-us/edge/download/insider?platform=linux